extensions/trello/src/utils/fetchTodos.tsx

```diff
@@ -0,0 +1,40 @@
+import { getPreferenceValues, showToast, ToastStyle } from "@raycast/api";
+import fetch from "node-fetch";
+import { TrelloFetchResponse } from "../trelloResponse.model";
+
+export interface Preferences {
+  token: string;
+  apitoken: string;
+  username: string;
+}
+
+const { token }: Preferences = getPreferenceValues();
```

Comment:
Hey @ChrisChinchilla – some optimization for a future version. You can destructure this in one go: const { token, apitoken, username } = ...